apps/src/netsim/NetSimVisualization.js

```diff
@@ -287,6 +309,97 @@ NetSimVisualization.prototype.onWireTableChange_ = function (rows) {
 };
 
 /**
+ * Based on new connectivity information, recalculate which 'fake' connections
+ * we need to display to show all nodes in a 'room' having direct wires to
+ * one another.
+ * @private
+ */
+NetSimVisualization.prototype.updateBroadcastModeWires_ = function () {
+  // Kill all fake wires
+  this.elements_.forEach(function (vizElement) {
+    if (vizElement instanceof NetSimFakeVizWire) {
+      vizElement.kill();
+    }
+  }, this);
+
+  // Generate new wires where they don't already exist
```

Comment:
Nice catch.  That comment is an artifact of me writing a more complex update that calculated the needed delta, then deciding it would be cleaner (for now) if the fake wires were just destroyed and recreated.